lib/test.js

```diff
@@ -230,33 +251,23 @@ Test.prototype._publicApi = function () {
 	api.plan = this.plan.bind(this);
 
 	function skipFn() {
-		self._assert();
+		self._assert(Promise.resolve());
```

Comment:
I think we should keep that here, as it makes it more obvious what we're doing, and it makes it a ***lot*** easier to reason about `this.assertions`.